config/notifies.go

```diff
@@ -120,7 +126,7 @@ func (c *EmailConfig) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	}
 	c.Headers = normalizedHeaders
 	if _, ok := c.Headers["Subject"]; !ok {
-		c.Headers["Subject"] = DefaultEmailSubject
```

Comment:
This change will mean that if the user specifies a subject with a case other than `Subject`, it'll break in the for loop above. It's messy, but the reason I did things this way.